src/Zonotope.jl

```diff
@@ -684,3 +685,36 @@ function translate(Z::Zonotope{N}, v::AbstractVector{N}; share::Bool=false
     generators = share ? Z.generators : copy(Z.generators)
     return Zonotope(c, generators)
 end
+
+"""
+    overapproximate(Z::Zonotope)
+
+Reduce the order of a given Zonotope to 1
+
+### Input
+
+- `Z`     -- zonotope
+
+### Output
+
+A overapproximation of the given zonotope using a Parallelotope.
```

Comment:
```suggestion
An overapproximation of the given zonotope using a parallelotope.
```
This description differs from the first sentence.